.circleci/smoke.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+set -e
+
+TMPDIR="$(mktemp -d 2>/dev/null || mktemp -d -t 'apiejs-smoketest')"
+API_ELEMENTS="$(pwd)"
+
+create_package() {
+  cd "$TMPDIR" && npm init --yes && cd "$API_ELEMENTS"
+}
+
+install() {
+  PACKAGE="$1"
+
+  TARBALL="$(cd packages/$PACKAGE && npm pack | tail -n1)"
+  mv "packages/$PACKAGE/$TARBALL" "$TMPDIR"
+
+  cd "$TMPDIR" && npm install "$TARBALL" && cd "$API_ELEMENTS"
+}
+
+echo "Setting up Fury in $TMPDIR"
+
+create_package
+install fury
+install fury-adapter-apib-parser
+install fury-adapter-oas3-parser
+install fury-adapter-apiary-blueprint-parser
+install fury-adapter-apib-serializer
+install fury-cli
+
+cd "$TMPDIR"
+npx fury --help
+npx fury --version
+
+# Parse Apiary Blueprint
+cat << EOF | npx fury -
+HOST: http://www.google.com/
+
+--- Sample API v2 ---
+---
+Welcome to the our sample API documentation. All comments can be written in (support [Markdown](http://daringfireball.net/projects/markdown/syntax) syntax)
+---
+EOF
+
+# Parse API Blueprint
+cat << EOF | npx fury -
```

Comment:
Do you want to assert somehow that the output is valid JSON/YAML as opposed to e.g. an exception? Even though... exception is non-zero exit code, so the script would halt - okay then.

Reply:
Yes, if there is any exception then Node would exit 1.

It would be neat to be able to assert that the result is valid JSON. Perhaps even that it is an object which has `element` of `parseResult` etc. I don't think it is that trivial to add though, I don't think it's possible to do it in a one-liner with node. In Python we could do `python -c 'import json, sys; print(json.load(sys.stdin))'` (this is Node image without Python).

Reply:
You redirect output to file, then cat the file (for a reference), and then something like

```bash
node -e 'JSON.parse(require("fs").readFileSync("foo"))'
```

Reply:
Forgot encoding, needs either toString() or readFileSync("foo", "utf8")